.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I would remove this setting as this is a subjective change.

Reply:
I got the config from https://github.com/brains-and-beards/mimic-server/blob/master/.eslintrc.js#L22.

Just added the prettierrc with those rules as saving on my editor format the file and then eslint complained (prettier rule) about it.

Reply:
Also i would switch Eslint rules off https://prettier.io/docs/en/integrating-with-linters.html but this is actually a subjective change.

Reply:
I'm actually the other way around, I prefer that the prettier rules are enforced by ESLint. Don't see a good reason for running two tools if they integrate nicely into one.